### ISSUE TITLE
Use accessible header for all public tables

### ISSUE THREAD

**Issue by roman-yagodin** [roman-yagodin/R7.University#96]
All public tables should have accessible headers with `<th scope="col">` and `<thead>` sections. This can be implemented by setting `UseAccessibleHeader` GridView attribute and specifying right table section for header row, like this: `grid.HeaderRow.TableSection = TableRowSection.TableHeader`.
